demos/gptj_causal_lm/python/requirements.txt

```
@@ -0,0 +1,5 @@
+transformers
+transformers[onnx]
+onnxruntime
+torch
+ovmsclient==2022.2
```

```suggestion
ovmsclient==2022.2
```
Shouldn't we have here 2022.3? @mzegla

---

ovmsclient 2022.3 is already on develop, but not yet published on PyPi. 
It's planned to publish along with the next minor release, so now pip install ovmsclient==2022.3 will not work yet. 
Perhaps we could drop specifying version in that case, so it's latest by default?

---

Will drop specific version